libs/translations/src/translations.service.ts

```diff
@@ -1,57 +1,61 @@
 import { Injectable } from '@nestjs/common';
-import { ConfigService } from '@backend/config';
 import { LoggerService } from '@backend/logger';
-import { LanguageCode, Translation } from './translations-model';
+import { Translation } from './translations-model';
 import { TranslationsRedisService } from './services/translations-redis.service';
-import { crowdinSourceStrings } from './data/source';
-import AdmZip from 'adm-zip';
 import Axios from 'axios';
-import fs from 'fs';
 import { ExceptionsService } from '@backend/exceptions';
+import { englishStrings } from './data/english-strings';
+import { supportedTranslations } from './data/supported-translations';
 
 @Injectable()
 export class TranslationsService {
   constructor(
-    private config: ConfigService,
     private logger: LoggerService,
     private exceptions: ExceptionsService,
     private translationsRedis: TranslationsRedisService
   ) {}
 
-  private fileName = 'all.zip';
-
   public async get(args: {
     languageCode: string;
     variables: {
       name: string;
     };
   }) {
-    if (!args.languageCode) {
-      const translation = { ...crowdinSourceStrings };
+    const languageCode = supportedTranslations.find(l =>
+      [l.locale, l.crowdinCode].includes(args.languageCode)
+    )?.locale;
+
+    if (!languageCode) {
```

Comment:
Don't you have the same issue than before here?
If anything is wrong in your code after this if, then the notification will crash.

why don't you put everything in a try catch + report error